cuml/rf_demo.ipynb

```diff
@@ -214,25 +214,25 @@
        "[5 rows x 29 columns]"
       ]
      },
-     "execution_count": 2,
+     "execution_count": 19,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "col_names = ['label'] + [\"col-{}\".format(i) for i in range(2, 30)]\n",
     "dtypes_ls = ['int32'] + ['float32' for _ in range(2, 30)]\n",
-    "data = cudf.read_csv('/datasets/higgs/HIGGS.csv', names=col_names, dtype=dtypes_ls)\n",
+    "data = cudf.read_csv('/datasets/higgs/HIGGS.csv', names=col_names, dtype=dtypes_ls, nrows=1_050_000)\n",
```

Comment:
I think that if there is an option to check if the dataset is downloaded on the system or not and then downloading if required would be nice to have in the notebook.